CMakeLists.txt

```diff
@@ -190,6 +208,18 @@ mark_as_advanced( KWIVER_INSTALL_SET_UP_SCRIPT )
 if( KWIVER_INSTALL_SET_UP_SCRIPT )
   install( PROGRAMS   ${KWIVER_SETUP_SCRIPT_FILE}
     DESTINATION ${CMAKE_INSTALL_PREFIX} )
+  if(WIN32)
+    install( PROGRAMS   ${KWIVER_SETUP_BATCH_FILE}
+      DESTINATION ${CMAKE_INSTALL_PREFIX} )
+  endif()
+endif()
+
+if ( fletch_DIR )
+  if(WIN32)
+    file( APPEND "${KWIVER_SETUP_BATCH_FILE}" "set PATH=%PATH%;${fletch_DIR}/bin\n" )
+    file( APPEND "${KWIVER_SETUP_BATCH_FILE}" "set PATH=%PATH%;${fletch_DIR}/x64/${_vcVersion}/bin\n" )
```

Comment:
Not a big deal, but you could do this in one call, right?  

    "set PATH=%PATH%;${fletch_DIR}/bin;${fletch_DIR}/x64/${_vcVersion}/bin\n"

Possibly this is two lines to make it more human readable?